.circleci/config.yml

```
@@ -151,8 +151,12 @@ jobs:
             fi
 
             ./node_modules/.bin/firebase use $ENV --token "$FIREBASE_TOKEN"
-            ./node_modules/.bin/firebase deploy --only functions --token "$FIREBASE_TOKEN"
+
+            # -f, --force: delete Cloud Functions missing from the current working directory without confirmation
+            ./node_modules/.bin/firebase deploy --only functions --token "$FIREBASE_TOKEN --force"
```

Should the `force` be outside the brackets ""?
```suggestion
            ./node_modules/.bin/firebase deploy --only functions --token --force "$FIREBASE_TOKEN"
```

---

Yup, it absolutely should be outside them

---

```suggestion
            ./node_modules/.bin/firebase deploy --only functions --token "$FIREBASE_TOKEN" --force
```

---

fixed in https://github.com/sparkletown/sparkle/pull/1436/commits/65bb018913e1588c73580c13b2acbcfaba590e7a